Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -19,12 +19,11 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher
 {
     public class SearchSkimmer : Skimmer<AnalyzeContext>
     {
-        public const string AssetFingerprint = "AssetFingerprint/v1";
-        public const string ValidationFingerprint = "ValidationFingerprint/v1";
-        public const string ValidationFingerprintHash = "ValidationFingerprintHash/v1";
+        public const string SecretHashSha256 = "SecretHashSha256/current";
```

Comment:
>SecretHashSha256 [](http://example.com/codeflow?start=28&length=16)

If this is a secret hash using sha256, then, we wouldn't require the `/current`, because sha256 won't change.

Unless we rename this to SecretHashCurrent, which in this case would be a sha256